platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/ConfigurationFileProxy.java

```diff
@@ -40,14 +39,21 @@
 
     public static final String DEFAULT_SOLR_CONFIG_PARENT_DIR = "etc";
 
-    public static final String SOLR_CONFIG_LOCATION_IN_BUNDLE =
-            new AbsolutePathResolver("solr/conf").getPath();
-
     public static final String DEFAULT_SOLR_DATA_PARENT_DIR =
             new AbsolutePathResolver("data/solr").getPath();
 
     public static final String CATALOG_SOLR_COLLECTION_NAME = "metacard";
 
+    protected static final List<String> SOLR_CONFIG_FILES =
```

Comment:
The `ConfigurationFileProxy` is usually not constructed in blueprint.  Currently only the embedded provides constructs it in blueprint.  If it was constructed in blueprint, this list of files would need to be copied everywhere a Solr client is used which would be several places.